test/utils/tools.js

```diff
@@ -39,8 +39,9 @@ function appStart() {
 // }
 
 function appStop() {
-	if (this.app && this.app.isRunning()) {
-		return this.app.stop();
+	if (app && app.isRunning()) {
+		app.stop();
+		return app.stop();
```

Comment:
why twice?

Reply:
@sk91 first time close is called it will show the confirmation screen. So we need to call it again so it will be closed and test will finish smoothly. That should also solve the problem with the windows build. Cheers

Reply:
👍